docker-compose.yml

```diff
@@ -230,7 +236,8 @@ services:
     # postgresql process to stop and restart.  Meanwhile, the unleash container tries to connect,
     # fails, and doesn't start.  So we add this shim script to wait until the connection is stable.
     # The script itself is sourced from the https://github.com/Unleash/unleash-docker container
-    command: ["/bin/sh", "/unleash/wait-for", "db:5432", "--", "node", "index.js"]
+    command: ["node", "index.js"]
```

Comment:
What allows us to remove the usage of this script? If it is indeed Ok to remove, please remove the comment.